### PR TITLE
Update some of the observability docs to respect the new structure

### PR DESCRIPTION
**Pull request by QuentinBisson**
### What this PR does / why we need it

Closes https://github.com/giantswarm/giantswarm/issues/29561
Closes https://github.com/giantswarm/giantswarm/issues/29566
Closes https://github.com/giantswarm/giantswarm/issues/29568

### Things to check/remember before submitting

- If you made content changes

    - Run `make dev` to render and proofread content changes locally.
    - Bump `last_review_date` in the front matter header if you reviewed the entire page.

### CODE REVIEW

File: src/content/advanced/observability/visualization/_index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+linkTitle: Visualization
+title: Visualization
+description: Guide explaining how to interact with our installation level grafana (how to access grafana, how to explore logs and metrics).
```
Comment (stone-z) [inline src/content/advanced/observability/visualization/_index.md:4]: I'd suggest using a different description than the `General` page

**Review by stone-z — APPROVED** [review]

File: src/content/advanced/observability/visualization/_index.md (hunk 1 above)
Comment (QuentinBisson) [reply src/content/advanced/observability/visualization/_index.md:4]: I agree but i'm not sure what to use 😂

Comment (QuentinBisson) [reply src/content/advanced/observability/visualization/_index.md:4]: I'm open to any suggestions

Comment (stone-z) [inline src/content/advanced/observability/visualization/_index.md:4]: I took a crack at it :smile: 
```suggestion
description: Advanced usage and customization guide for Giant Swarm platform observability features.
```

Comment (QuentinBisson) [reply src/content/advanced/observability/visualization/_index.md:4]: It's looking great :) thanks

**Pull request merged**